cmd/configure.go

```diff
@@ -199,9 +199,10 @@ func configureTarget(cmd *cobra.Command, args []string) error {
 
 	var targetName string
 	var target *workflow.Target
+	var targetConfig *config.Configuration
 	if existingTarget == "" {
 		// If we add multiple targets to one workflow file the out dir of a target cannot be the root dir
-		if err := prompts.MoveOutDir(workflowFile, existingTargets); err != nil {
+		if err := prompts.PromptForOutDirMigration(workflowFile, existingTargets); err != nil {
```

Comment:
Don't exactly follow this logic. Why do we only migrate when there _isn't_ an existing config?

Reply:
Maybe I can word the comment better. If you want to add a second config to an existing workflow file you must change the outdir of the existing target to not be the root directory.